zerver/openapi/zulip.yaml

```diff
@@ -5502,25 +5502,33 @@ paths:
       summary: Send a message
       tags: ["messages"]
       description: |
-        Send a stream or a private message.
+        Send a [stream](/help/streams-and-topics) or
+        a [direct](/help/direct-messages) message.
```

Comment:
nit, and I see this is a pre-existing issue, but since we're looking at this sentence: this wording parses much more strongly for me as "send a stream, or send a direct message" than as "send a stream message, or send a direct message".

Probably better to just say "Send a [stream message](…) or a [direct message](…)".

Reply:
Yeah that's a good improvement.

Reply:
Agreed. Updated in current version.